src/views/v2/components/loader/loader.php

```diff
@@ -14,14 +14,18 @@
  * @version 5.0.4
  */
 
-$classes = $this->get( 'classes' ) ?: [];
+$classes        = $this->get( 'classes' ) ?: [];
+$visible        = tribe_is_truthy( $this->get( 'visible' ) ) || false;
 
 $spinner_classes = [
 	'tribe-tickets-loader__dots',
 	'tribe-common-c-loader',
-	'tribe-common-a11y-hidden',
```

Comment:
Why are we modifying the default behavior of the loader? The norm is that by default it's hidden. This change can potentially cause issues in other places where we use the loader. 

I feel that if we actually need this to be visible by default in a particular scenario we can do it with the JS method directly.

Reply:
We are not getting into the ready state while the PayPal Checkout script is getting loaded. So, it keeps showing blank and by the time we trigger the loader to show, the Paypal buttons are already rendered and the loader doesn't help at all.

So, I have modified the loader to be visible at page render immediately while the JS scripts get loaded.

The loader template should be unaffected as we are changing the classes only if the `visible` variable is set.

Reply:
I would still avoid changing the default template and make the changes by overwriting this from the Usage of the component

Reply:
This change shouldn't affect the other uses of the loader already? Since we are not passing in a `visible` property to the other loaders, it should still print the hidden class.

However, I do think that adding the hidden class by default is not the right call here, we should allow users of this template to pass it in on their own (there is now no way to show by default). Is there a way we can change that in the places we need to? Or does that touch too many pieces?

Reply:
> Is there a way we can change that in the places we need to? Or does that touch too many pieces?

At least the tickets block, RSVP, AR page, AR modal, Manual Attendees.

With regards to the default behavior (hidden), ET is following [what TEC does](https://github.com/the-events-calendar/the-events-calendar/blob/master/src/views/v2/components/loader.php#L17), so that's kind of in-line.

Reply:
Yeah, not the best implementation :(, we hadn't thought about that.

Reply:
The other thing we could do is in the JS, on ready, we can remove the `hidden` class.

Reply:
Yep, that's what I suggested initially, and here's what Rafsun found: https://github.com/the-events-calendar/event-tickets/pull/2079#discussion_r705233563